components/net/http_loader.rs

```diff
@@ -529,6 +530,25 @@ pub fn modify_request_headers(headers: &mut Headers,
     // https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch step 11
     if load_data.credentials_flag {
```

Comment:
xmlhttprequest.rs never sets with_credentials to true anywhere except for `SetWithCredentials`, which the test in `send-authentication-basic.htm` never called.